### PR TITLE
Add PulumiUP 2022 registration

### PR DESCRIPTION
**Pull request by zchase**
This PR adds the initial structure and content for the PulumiUP registration page.

### CODE REVIEW

File: themes/default/content/blog/announcing-pulumi-up-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Calling all cloud engineers! Today we announce the second annual [PulumiUP]({{< relref "/pulumi-up" >}}) virtual conference taking place on May 4, 2022. We will also host the inaugural Pulumi Community Summit on May 5 to virtually assemble the world’s cloud, infrastructure, and Pulumi practitioners of all experience levels (hint: that’s you --- all are welcome)! [Register for both events here]({{< relref "/pulumi-up" >}}).
+
+We’ve worked hard to curate a half-day of learning and expert insights from the people that are forming the best practices of the modern cloud and cloud native infrastructure era. On May 4, join us for four hours of exciting Pulumi product news and demos, technical talks, case studies of modern cloud innovators, and conversations on cloud topics that you won’t hear anywhere else. Hear from Pulumi’s CEO and CTO, pioneering engineers and engineering leaders from different industries, and Pulumi’s engineers. We are also introducing two tracks designed for Practitioners and Engineering Leaders.
+
+## Who’s speaking at PulumiUP 2022?
+
+* Aaron Friel, Staff Software Engineer, Pulumi
+* Elad Ben-Israel, Principal Software Development Engineer, Amazon Web Services
```
Comment (gthuang) [inline themes/default/content/blog/announcing-pulumi-up-2022/index.md:23]: @aaronkao is this correct title for Elad?

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 2]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="lg:h-64 p-6">
+                                <p class="my-0 mt-2">
+                                    30 min
+                                </p>
+                                <h5 class="mt-0 mt-4">
+                                    PulumiUP 2022 Keynote: The Future of Cloud Enabled IaC
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:118]:
```suggestion
                                    PulumiUP 2022 Keynote: The Future of Cloud-Enabled IaC
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 3]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+
+                            <div class="lg:h-64 p-6">
+                                <p class="my-0 mt-2">
+                                    60 min
+                                </p>
+                                <h5 class="mt-0 mt-4">
+                                    PulumiUP 2022 Keynote: Cloud Migration and Multicloud Solutions Forecast
+                                </h5>
+                                <p>
+                                    Learn about “multi cloud solutions yadda yadda and yadda super insightful forecast with
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:163]: Are we intending to leave the `yadda yadda...` part?

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 4]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                                    thought leadership stuff” with Joe Dufy and Luke Hoban.
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:164]:
```suggestion
                                    thought leadership stuff” with Joe Duffy and Luke Hoban.
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 5]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+            Get ready to be inspired, challenged, and transformed. Learn from experts and success stories about
+            early adopters.
+        </p>
+
+        <div class="flex flex-col lg:flex-row">
+            <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                {{ partial "color-icon.html" (dict "icon" "lightning" "icon_color" "yellow") }}
+                <p class="key-feature-description">
+                    Learn from cloud engineering experts about how they manage and solve frustrations
+                    with handling cloud complexity all the way from cloud native applications to structure
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:186]: I'm not quite sure what this sentence is saying (are we learning about managing frustrations around cloud complexity from nothing to cloud native and then about structuring engineering orgs?), so I took a crack at making it make more sense.
```suggestion
                    with handling cloud complexity all the way from cloud native applications to structuring
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 6]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                    Unlock the power of the cloud with thought-provoking presentations that are
+                    designed to help you transform your organization, your business, and yourself.
+                    Access open source code on Github and get involved with the community.
+                </p>
+            </div>
+
+            <div class="w-full lg:w-1/3 px-8 mt-8 lg:mb-0">
+                {{ partial "color-icon.html" (dict "icon" "rocketship" "icon_color" "salmon") }}
+                <p class="key-feature-description">
+                    You will learn how top performing cloud engineering teams and organizations are leveraging
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:203]:
```suggestion
                    You will learn how top-performing cloud engineering teams and organizations are leveraging
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 7]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+            <div class="max-w-3xl card white-background mx-auto p-6 text-left">
+                <h3>Who Should Attend</h3>
+                <p>
+                    Business leaders, engineering managers, developers, and anyone who is interested in learning
+                    more about the cloud and cloud engineering.
+                </p>
+
+                <ul>
+                    <li>CIOs, CTOs, CDOs, CXOs, CISOs, VPs, SVPs, and CPOs.</li>
+                    <li>CEOs, COOs, CMOs, andC FOs leading digital initiatives.</li>
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:230]:
```suggestion
                    <li>CEOs, COOs, CMOs, and CFOs leading digital initiatives.</li>
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 8]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                    <li>Directors and Heads of: IT, Infrastructure, Engineering, Architecture, and Innovation.</li>
+                    <li>Chief Engineers, Consultants and Digital Transformation Executives.</li>
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:232]:
```suggestion
                    <li>Chief Engineers, Consultants, and Digital Transformation Executives.</li>
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 9]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+        <div class="flex flex-wrap my-8">
+            <div class="w-full lg:w-1/4 border-b lg:border-b-0 lg:border-r border-white">
+                <p class="text-3xl">Wednesday, May 4th</p>
+                <p>8:00AM - 12:00PM PDT<p>
+            </div>
+            <div class="w-full lg:w-3/4 lg:pl-6">
+                <p class="text-3xl">PulumiUP</p>
+                <p>
+                    PulumiUP is for anyone who is interested in cloud engineering, cloud infrastructure,
+                    software development, modern applications or Pulumi. Whether you're a seasoned cloud
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:281]:
```suggestion
                    software development, modern applications, or Pulumi. Whether you're a seasoned cloud
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 10]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+        <div class="flex flex-wrap my-8">
+            <div class="w-full lg:w-1/4 border-b lg:border-b-0 lg:border-r border-white">
+                <p class="text-3xl">Thursday, May 5th</p>
+                <p>8:00AM - 12:00PM PDT<p>
+            </div>
+            <div class="w-full lg:w-3/4 lg:pl-6">
+                <p class="text-3xl">Community Summit</p>
+                <p>
+                    Join your friends in the Pulumi community for the first ever Pulumi Community Summit.
+                    This virtual event will include an “Open Spaces'' concept where you can propose and
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:297]:
```suggestion
                    This virtual event will include an "Open Spaces" concept where you can propose and
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 11]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                    vote on topics ahead of time. Those topics with the most interest will form the agenda
+                    for the day, with virtual break out rooms centered around those themes.
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:299]:
```suggestion
                    for the day, with virtual breakout rooms centered around those themes.
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 12]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                </event-session-registration-modal>
+            </div>
+        </div>
+    </section>
+
+    <section id="ctas" class="container mx-auto py-16">
+        <div class="flex flex-wrap items-stretch">
+            <div class="w-full lg:w-1/2 p-3">
+                <div class="h-full card white-background p-6">
+                    <h3>Connect wit Peers on GitHub</h3>
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:335]:
```suggestion
                    <h3>Connect with Peers on GitHub</h3>
```

File: themes/default/layouts/page/pulumi-up-2022.html [hunk 13]
```diff
@@ -28,35 +30,321 @@ <h1>The Cloud Awakens</h1>
...
+                    <p class="mb-8">Pulumi open source code is available to all. See what we are working on and interact with the community.</p>
```
Comment (nimbinatus) [inline themes/default/layouts/page/pulumi-up-2022.html:336]: Just a quick adjustment to make it more inclusive.
```suggestion
                    <p class="mb-8">Pulumi open source code is available to all. Explore what we are working on and interact with the community.</p>
```

File: themes/default/content/pulumi-up/_index.md [hunk 14]
```diff
@@ -10,4 +10,69 @@ event_date: 2022-05-04T09:00:00-07:00
...
+    - name: mercedes-benz
+      link: /case-studies/mercedes-benz
+    - name: sans
+      link: /case-studies/sans-institute
+    - name: snowflake
+      link: /case-studies/snowflake
+    - name: credijusto
+      link: /case-studies/credijusto
+    - name: skai
+      link: /blog/kenshoo-migrates-to-aws-with-pulumi
```
[the remark this answers is not in the document]
Comment (zchase) [reply themes/default/content/pulumi-up/_index.md:60]: Yeah Kenshoo became Skai

**Review by nimbinatus — APPROVED** [review]
+1. As long as the build goes through, :shipit:

**Pull request merged**